docs/advanced.md

```diff
@@ -44,6 +45,37 @@ which can be used to show custom page numbers or add other elements to the UI.
 - `step`: (number) the current visible step in an Appear or Step component
 - Each slide includes a `meta` object with a `notes` field when the Notes component is used within a slide
 
+#### Example
+
+The example below will display the current slide out of the total amount of slides.
+
+```js
+import React from 'react'
+import { css } from '@emotion/core'
+
+function AtTheBottomCenter ({ children }) {
+  const css = {
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    bottom: 0,
+    color: #ffffff;
+    textAlign: 'center',
+  }
+
+  return <div css={ css }>
+    { children }
```

Comment:
Seems like prettier isn't formatting this as expected. I'll clean up the syntax after merging